datadog/cmd/main/main.go

```diff
@@ -226,30 +232,52 @@ func (d *DatadogCostSource) getDDCostsForWindow(window opencost.Window, listPric
 		ccResp.Errors = append(ccResp.Errors, err.Error())
 	}
 
+	previousChargeCosts := make(map[string]float32)
+
+	isDaily := window.Duration().Hours() == 24
```

Comment:
This feels like a pretty weak condition to me, particularly since it is leading to a hardcoded division. What do you think about something like this?
```suggestion
	switch window.Duration().Hours() {
	case 24:
	    costFactor = 1.0 / 24.0
	case 1:
	    costFactor = 1
	default:
	    ... error
	}
```

Reply:
I think that's a much stronger condition, I appreciate the suggestion.